README.md

```diff
@@ -244,6 +300,18 @@ TSDB or Time Series Databases are software systems for storing and retrieving ti
 
 A trace is the complete journey of a request or workflow as it moves from one part of the system to another. It is achieved by adding a standard trace id as the request/action flows through all the hops.
 
+### Tail Sampling
+
+Tail sampling is a technique for collecting metrics data from the slowest requests in a system. This can be useful for understanding the causes of tail latency and identifying performance bottlenecks.
+
+### Tail Latency
+
+Tail latency refers to the latency of the slowest requests in a system. In SRE, tail latency is an important metric to monitor, as it can have a significant impact on the user experience and overall system performance.
```

Comment:
```suggestion
Tail latency refers to the latency of the slowest requests in a system. Tail latency is an important metric to monitor, as it can have a significant impact on the user experience and overall system performance.
```